static/external/manyplot.js

```diff
@@ -239,6 +240,25 @@ function createPlotter( canvas, multiFrame, options ) {
         }
     };
 
+    plotter.sortData = function (dataPairs) {
+        dataPairs.sort((a, b) => {
+            // Sanity check that the pairs contain data
+            if (a.yData && a.yData.name && b.yData && b.yData.name) {
+                var nameA = a.yData.name.toUpperCase(),
+                    nameB = b.yData.name.toUpperCase();
+                if (nameA < nameB) {
```

Comment:
Speaking of obscure cases... I suppose a user could elect to name sequences with purely numeric values in ascending order (i.e., 1,2,3,4....10,11,12) in which case our sort would result in "10" coming before "2".  But this seems highly unlikely to happen in practice.  Regardless, I think making sure we have a consistent order is far more important than anything else.